web/templates/support/index.html.eex

```diff
@@ -1,203 +1,151 @@
 
   <div class="hl-bg-grey pa3">
-    <h3 class="tc">Free 24-hour support services</h3>
+    <h3 class="tc">In an emergency</h3>
     <div class="tc">
       <%= img_tag("/images/Get_Support.png", class: "w-20 tc") %>
     </div>
 
-    <h4>Local support</h4>
-    <p><strong>South London and Maudsley NHS Foundation Trust mental health support line </strong>
-    <%= link_to("0800 731 2864", to: "tel:08007312864") %> for anyone in Southwark, Lambeth, Lewisham or Croydon.
-    Calls will be answered by a nurse who can give advice on mental health, medication and accessing services</p>
-
-    <p><strong>Solidarity in a crisis</strong>
-      <%= link "0300 123 1922",
-      to: "tel:03001231922", class: "hl-aqua flex-wrap underline" %>, or text
-      <%= link "07595864103", to: "tel:07595864103",
-      class: "hl-aqua flex-wrap underline" %> or email
-      <%= link "outofhours-solidarity@certitude.org.uk",
-      to: "mailto:outofhours-solidarity@certitude.org.uk",
-      class: "hl-aqua flex-wrap underline" %>.
-      Free emotional support for people living in Lambeth, Southwark and Lewisham.
-      The service is run by peer supporters who have their own experience of facing and overcoming crisis.
-      Mon to Fri 6pm – 12 midnight, and Sat &amp Sun 12 midday – 12 midnight</p>
-
-    <h4 class="mt-3rem">National support</h4>
-    <p><strong>The Samaritans</strong> <%= link "116 123", to: "tel:116123", class: "hl-aqua flex-wrap underline" %>
-      is a national service (and you don't have to be suicidal to call for support)</p>
-
-    <p><strong>Childline</strong>
-      <%= link "0800 1111", to: "tel:0800 1111",
-      class: "hl-aqua flex-wrap underline" %> or
-      <%= link "chat online to a counsellor",
-      to: "https://www.childline.org.uk/get-support/1-2-1-counsellor-chat/",
-      class: "hl-aqua flex-wrap underline" %>.
-      A free confidential line for children &amp young people (you don’t have to give your name if you don’t want to)</p>
-
-    <p><strong>FRANK</strong> <%= link "0300 123 6600", to: "tel:0300 123 6600",
-      class: "hl-aqua flex-wrap underline" %>
-      A free confidential helpline for anyone concerned about drug use.
-      <%= link "Chat online 2-6pm",
-      to: "http://www.talktofrank.com/livechat",
-      class: "hl-aqua flex-wrap underline" %> or read
-      <%= link "what to do", to: "http://www.talktofrank.com/emergency-help",
-      class: "hl-aqua flex-wrap underline" %>
-      if a friend or relative is having a bad reaction to a drug</p>
-
-    <p><strong>NHS</strong> <%= link_to("111", to: "tel:111") %> call for urgent, but non-emergency advice</p>
-
-    <p><strong>Emergency services</strong> <%= link_to("999", to: "tel:999") %>
-      or find your nearest <%= link_to("A&E", to: "http://www.nhs.uk/Service-Search/Accident-and-emergency-services/LocationSearch/428") %></p>
-  </div>
+    If you have immediate concerns about your physical safety and/ or wellbeing, or the physical safety and
```

Comment:
looks like this sentence is meant to be in a set of p tags `<p>` `</p>`. Starting with "If you have immediate concerns... and ending with "the emergency services:"